gatsby-config.js

```diff
@@ -5,21 +5,6 @@ module.exports = {
   },
   plugins: [
     'gatsby-plugin-react-helmet',
```

Comment:
Do the Google fonts still work without the `gatsby-plugin-prefetch-google-fonts`? How are they loaded now?

Reply:
I deleted it because I found this issue. But now I added the replacement: gatsby-plugin-google-fonts